C++/Env/Environment.h

```diff
@@ -0,0 +1,14 @@
+#ifndef _ENVIRONMENT_H_
+#define _ENVIRONMENT_H_
+
+#include <string>
+
+class Environment
+{
+public:
+  virtual std::string getEnvironment (void) = 0;
+  
+private:
```

Comment:
No we cant.  How else would you like to call this method on the derived classes?